Firestore/Example/Tests/Local/FSTLocalStoreTests.mm

```diff
@@ -148,8 +150,8 @@ - (void)applyRemoteEvent:(const RemoteEvent &)event {
   _lastChanges = [self.localStore applyRemoteEvent:event];
 }
 
-- (void)notifyLocalViewChanges:(FSTLocalViewChanges *)changes {
-  [self.localStore notifyLocalViewChanges:@[ changes ]];
+- (void)notifyLocalViewChanges:(LocalViewChanges)changes {
+  [self.localStore notifyLocalViewChanges:std::vector<LocalViewChanges>{changes}];
```

Comment:
Nit: you can `std::move` `changes` into the vector.

Reply:
Done.